tests/topotests/grpc_basic/test_basic_grpc.py

```diff
@@ -108,7 +108,7 @@ def test_capabilities(tgen):
     logging.debug("grpc output: %s", output)
 
     modules = sorted(re.findall('name: "([^"]+)"', output))
-    expected = ["frr-backend", "frr-interface", "frr-routing", "frr-staticd", "frr-vrf"]
+    expected = ["frr-backend", "frr-host", "frr-interface", "frr-logging", "frr-routing", "frr-staticd", "frr-vrf", "ietf-srv6-types", "ietf-syslog-types"]
     assert modules == expected
```

Comment:
so maybe this test is off-target a bit: maybe the test should be that some expected values are present, but not necessarily that there are no additional, new values?

Reply:
Hmm I think it's fine, the test is called `basic_grpc` for a reason, it just checks that all the capabilities are there and a basic request returns some default config. I mean we could add a test-case for every single capability, but I don't think that's really worth the lines of code introduced.

Reply:
AFAICT this is not configurable and defined here:
https://github.com/FRRouting/frr/blob/10b09c3db4e8826d625671d3493bc7e33389f315/mgmtd/mgmt_be_adapter.c#L165-L171
?